Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -5,6 +5,7 @@
 #import <UIKit/UIWindow.h>
 
 #import "MSAlertController.h"
+#import "MSPerformSelectorUtil.h"
```

Comment:
It's Vendor folder, it shouldn't depend on SDK utils.
I suggest just move this `MSAlertController` to core module, because it's our helper, not 3rd-party thing

Reply:
Moved